cypress/support/commands.ts

```diff
@@ -234,6 +234,24 @@ export const isIoxOrg = (): Cypress.Chainable<boolean> => {
   })
 }
 
+const IOX_SWITCHOVER_CREATION_DATE = '2023-01-31T00:00:00Z'
+
+export const mockIsCloud2Org = (): Cypress.Chainable<null> => {
+  const beforeCutoff = new Date(
+    new Date(IOX_SWITCHOVER_CREATION_DATE) - 10000
+  ).toISOString()
```

Comment:
This mocking produces an older tsm or iox account. Bescially, our full supported feature set prior to our next (cloud 3? 2.5?) release iterations.

Probably could do with better namings. Right now, this only impacts deprecations with older versus newer iox accounts. Eventually could also apply to tsm accounts if we do staged deprecations prior to migration to iox.  Either way, I thought "Cloud2" was a good name. Maybe. 🤷🏼‍♀️